SilverSolver/assignment4/islands.py

```diff
@@ -0,0 +1,44 @@
+from collections import deque
+from typing import List
+
+def neighbourhood(i: int, j: int, islandMatrix: List[List[int]]) -> List[List[int]]:
+    # helper function, returns the neighbourhood of (i, j) cell in islandMatrix
```

Comment:
nit: turn the comment into a Python docstring:
"helper function, returns the neighbourhood of (i, j) cell in islandMatrix."

A Python docstring is technically the first line after the declaration and it must be a literal string.